lib/code_corps/endpoint.ex

```diff
@@ -38,5 +38,6 @@ defmodule CodeCorps.Endpoint do
     key: "_code_corps_key",
     signing_salt: "RJebPcfU"
 
+  plug Corsica, origins: "http://localhost:4200"
```

Comment:
The allowed origins should probably be defined in the env config files and hen accessed here. I think something like:
```elixir
config :code_corps, allowed_origins: {"http://localhost:4200"} 
```

in config/dev.ex and then accessed here like
```elixir
Application.get_env :code_corps, :allowed_origins
```
would do the trick